### ISSUE TITLE
NFS6 out of scope?

### ISSUE THREAD

**Issue by hazelnot** [OpenNFS/OpenNFS#23]
This isn't an issue per se, but I feel like Hot Pursuit 2 is a bit out of scope, and would better fit into a project aiming to recreate the Black Box engine. HP2 was the first NFS game made by Black Box, and they used that engine for every Need for Speed game they worked on except The Run.

**Comment by AmrikSadhra** [comment]
I understand where you're coming from, but where HP2 'support' is promised, it is purely for the loading of its vehicle and track assets, not a complete engine overhaul into Blackbox structure for ONFS.

**Comment by Keiiko** [comment]
It is not about what engine who used. It is about the era. ONFS is about classic NFS titles where HP2 surely belongs, but not UG and beyond.

When we look at commom elements of classic nfs titles then we see that there are diffefent maps (that are mostly devided into two tracks with common part), cars that you can change its color or tune its performance in a way. NFS3-5 have car addons. All of classic nfs follow simple rule - to win the game you need to race this set of races in order to win track/car.

There we can say that it is the same as in UG, UG2, MW, .... Yes, but there are major changes. Now there is only one city. UGs are in night only. You can tune the car as you want, color, stickers, body parts, neons, nitro, etc. You know what i mean. Then the music is different. Now there is also bigger story. You have tasks to race somewhere on time, race against guy on blacklist or just drive in the city from nowhere to nowhere.

Basically HP2 IS the last classic NFS era game but in the same time dont belong there technologically.

**Comment by hazelnot** [comment]
@Keiiko I disagree, cause a project like this should imo be based on the tech it aims to replicate, but then again, the most coding I can do is a bit of JavaScript so I can't tell people what to do lol. I think what @AmrikSadhra is fair though 😅.

**Comment by Keiiko** [comment]
@hazelnot ONFS is about reverse engineering original assets from nfs1-6 from different platforms... then it is about 3D rendering them and lastly about physics not about rewriting whole game from scratch to best mimic it.

Then again im still hoping that each game on each platform would be like a plugin that you can download load and import from iso/cd... Different physics or menu skins could be also inclided but it is agreed to make it look like nfs3pc as default (or close to) with menu be new but would be inspired by nfs3 (you really need it because it would need more things than original)

Panda can explain it better...

And about HP2 it would be just about importing maps and cars from there... like I said... belongs to the classic era.
